src/AsyncTCP.cpp

```diff
@@ -1,10 +1,25 @@
 // SPDX-License-Identifier: LGPL-3.0-or-later
 // Copyright 2016-2025 Hristo Gochkov, Mathieu Carbou, Emil Muratov
 
-#include "Arduino.h"
-
 #include "AsyncTCP.h"
 
+#include <esp_log.h>
+
+#ifndef CONFIG_ASYNC_TCP_NO_ARDUINO
```

Comment:
Arduino can be detected, because it adds it's own defines, so this could be made to not require any extra steps from the user. If `ARDUINO` is not defined, then it must be IDF build